js/build/shortcode-ui.js

```diff
@@ -1144,6 +1144,17 @@ var editAttributeField = Backbone.View.extend( {
 		}
 
 		this.$el.html( this.template( data ) );
+
+		// Ensure default value for select field.
+		if ( 'select' === data.type && '' === this.model.get( 'value' ) ) {
+			var firstVisibleOption = _.first( data.options );
+			if ( 'undefined' !== typeof firstVisibleOption.value ) {
+				this.setValue( firstVisibleOption.value );
+			}
+		}
+
+		this.$el.html( this.template( data ) );
```

Comment:
Where's this line coming from? It doesn't seem to be in the `src` file and it duplicates L1146 above...